bigchaindb/common/utils.py

```diff
@@ -92,9 +92,28 @@ def validate_all_keys(obj_name, obj, validation_fun):
     """
     for key, value in obj.items():
         validation_fun(obj_name, key)
-        if type(value) is dict:
+        if isinstance(value, dict):
             validate_all_keys(obj_name, value, validation_fun)
-    return
+
+
+def validate_all_value_for_key(obj, key, validation_fun):
+    """Validates value for all (nested) occurences of `key` in `obj`
```

Comment:
Should be "Validate", I'm not sure we documented this but we prefer imperative sentences. See also [Good Python docstrings by example](http://blog.dolphm.com/pep257-good-python-docstrings-by-example/).